### PR TITLE
Add French README page

### PR DESCRIPTION
**Pull request by nejos97**
# Description

I've added the French README page and added the link in the other readme as well. I'm open to any other proposal if required. Thank you

### CODE REVIEW

File: README.ru.md [hunk 1]
```diff
@@ -52,11 +52,11 @@
 - [Содержание](#содержание)
 - [Вступление](#вступление)
 - [Особенности](#особенности)
-    - [Roadmap](#roadmap)
+  - [Roadmap](#roadmap)
 - [Быстрый старт](#быстрый-старт)
 - [Спонсоры](#спонсоры)
 - [FAQ](#faq)
-- [График звёздочек](#график-звёздочек-репозитория-относительно-времени)
```
Comment (leaanthony) [inline README.ru.md:59]: Why this change?

Comment (nejos97) [reply README.ru.md]: it was a correction from my editor!! I just fixed it.

**Review by leaanthony — COMMENTED** [review]
Thanks 🙏 Please could you add an entry to the changelog located at `website/src/pages/changelog.mdx`?

**Review by leaanthony — APPROVED** [review]

**Comment by leaanthony** [comment]
Merci beaucoup 🙏
